### PR TITLE
Shift most discussion of Workshops to /Guide

### PR DESCRIPTION
**Pull request by frivoal**
This PR consolidates the rules about Group meetings with those about Workshop and Symposia, and delegates other discussion and description of Workshops to /Guide.

This makes it clear that rules about minute taking or meeting recording apply to Workshops as well as Group meetings, regroups rules about advance notice on meetings to a single place, and shortens the Process by focusing it on enforceable rules, while letting /Guide provide more elaborate description and non-binding guidance.

See #419

***
<a href="https://pr-preview.s3.amazonaws.com/frivoal/w3process/pull/876.html" title="Last updated on May 22, 2024, 5:05 PM UTC (603f8ff)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/876/8b48f80...frivoal:603f8ff.html" title="Last updated on May 22, 2024, 5:05 PM UTC (603f8ff)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -519,10 +519,14 @@ Individuals Representing a Member Organization</h5>
...
-<h4 id="GeneralMeetings">
+<h4 id="GeneralMeetings" oldids="GAEvents, EventsS">
 Meetings</h4>
 
-	The requirements in this section apply to the official meetings of any [=W3C group=].
+	The requirements in this section apply to the official meetings of any [=W3C group=],
+	as well as to official W3C meetings with open-ended participation,
+	such as <dfn id="EventsW" lt="Workshop | W3C Workshop" export>Workshops</dfn>
+	(as described in the <a href="https://www.w3.org/Guide/meetings/workshops">Guide article about Workshops</a> [[GUIDE]])
+	or other events open to the whole [=Member|Membership=] or to the public.
```
Comment (fantasai) [inline index.bs:529]: Can we shorten this to

```suggestion
	such as <a id="EventsW" href="https://www.w3.org/Guide/meetings/workshops">Workshops</a>. [[GUIDE]]
```

?

Comment (fantasai) [reply index.bs:529]: Maybe with s/open-ended participation/open-ended participation from the Membership and/or the public/ if we want to be clear.

File: index.bs [hunk 2]
```diff
@@ -589,6 +593,10 @@ Meeting Scheduling and Announcements</h5>
 	about the date and location of a meeting.
 	Shorter notice for a meeting is allowed
 	provided that there are no objections from group participants.
+	In the case of meetings
+	whose participation is not tied to any pre-existing [=Group=],
+	such as [=Workshops=],
+	shorter notice is not allowed.
```
Comment (fantasai) [inline index.bs:599]: Do we want to forbid this completely, or allow for exceptions via e.g. approval of the CEO? I think we might want to allow exceptions...

File: index.bs [hunk 3]
```diff
@@ -519,10 +525,12 @@ Individuals Representing a Member Organization</h5>
...
 	(or group of <a href="#MemberRelated">related Members</a>).
 
-<h4 id="GeneralMeetings">
+<h4 id="GeneralMeetings" oldids="GAEvents, EventsS">
 Meetings</h4>
 
-	The requirements in this section apply to the official meetings of any [=W3C group=].
+	The requirements in this section apply to the official meetings of any [=W3C group=],
+	as well as to official W3C meetings with open-ended participation from the Membership and/or the public,
+	such as [=Workshops=] or other events open to the whole [=Member|Membership=] or to the public.
```
Comment (fantasai) [inline index.bs:533]: The text here is somewhat redundant.

Comment (frivoal) [reply index.bs]: Indeed. That was stray text left after a rewrite. Fixed.

File: index.bs (hunk 2 above)
Comment (frivoal) [reply index.bs]: As discussed on the call, I've modified the PR to keep only applying this stricter requirement to workshops themselves, and not to a generalized category of open meetings.

File: index.bs (hunk 1 above)
Comment (frivoal) [reply index.bs]: addressed by the rewrite done after last meeting

**Review by fantasai — APPROVED** [review]

**Review by chrisn — APPROVED** [review]

**Review by nyeinmaung-mm — APPROVED** [review]

**Comment by alaminsohagofficial** [comment]
ok
